doc/configure-device.md

```diff
@@ -4,19 +4,25 @@
 
 1. Make sure your host PC and TV are on the same network (connected to the same access point).
 
-1. Enable the developer mode.
+1. Enable the developer mode.(OS version: Tizen 9.0)
```

Comment:
Clarification: "The images below are from a Tizen 9.0 TV. They may look different on other TVs."